integrationTests/vm/arwen/testdata/erc20-c-03/wrc20_arwen.c

```diff
@@ -173,51 +163,43 @@ void transferToken() {
 
   // 2nd argument: amount (should not be negative)
   bigInt amount = bigIntNew(0);
-  bigIntGetUnsignedArgument(1, amount);
+  bigIntGetSignedArgument(1, amount);
```

Comment:
Shouldn't this be unsigned? Can also remove the negativity check below.

Reply:
See above.